src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -115,3 +115,32 @@ Variable SparqlQleverVisitor::visitTypesafe(
     SparqlAutomaticParser::VarContext* ctx) {
   return Variable{ctx->getText()};
 }
+
+// ____________________________________________________________________________________
+GraphPatternOperation::Values SparqlQleverVisitor::visitTypesafe(
+    SparqlAutomaticParser::InlineDataContext* ctx) {
+  return visitTypesafe(ctx->dataBlock());
+}
+
+// ____________________________________________________________________________________
+GraphPatternOperation::Values SparqlQleverVisitor::visitTypesafe(
+    SparqlAutomaticParser::DataBlockContext* ctx) {
+  if (ctx->inlineDataOneVar()) {
+    return GraphPatternOperation::Values{
+        std::move(visit(ctx->inlineDataOneVar()).as<SparqlValues>())};
```

Comment:
The return type here is redundant, so
`{std::move(visit(...))}` would already suffice instead of `Values{std::move(...`)
(same below).

but even better:
Also move the "visitTypesafe" one level down,
s.t. this becomes return {visitTypesafe(ctx->inlineDataOneVar());}
same for the "else".

Reply:
maybe the `{}` are even not needed, because there's only one argument to the constructor.